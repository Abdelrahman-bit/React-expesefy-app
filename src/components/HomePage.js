import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';


const HomePage = () => (
    <div>
        <p>wlecom in the expencive app "Home page"</p>
        <ExpenseListFilters />
        <ExpenseList />
    </div>
)

export default HomePage;
