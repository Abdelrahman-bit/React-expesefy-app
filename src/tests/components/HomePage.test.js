import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../../components/HomePage';

test('shold render the HomePage component', () =>{
    const wrapper = shallow(<HomePage />);
    expect(wrapper).toMatchSnapshot();
})