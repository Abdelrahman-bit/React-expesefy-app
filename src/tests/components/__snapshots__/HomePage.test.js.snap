// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shold render the HomePage component 1`] = `
<div>
  <p>
    wlecom in the expencive app "Home page"
  </p>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
